### PR TITLE
Fix: Linter fails (#1136)

### PR DESCRIPTION
**Pull request by pilinux**
**Describe the PR**
Fix: Linter fails (#1136)

### CODE REVIEW

File: swagger.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 )
 
 // Name is a unique name be used to register swag instance.
-const Name = "swagger"
+const Name = "Swagger"
```
Comment (ubogdan) [inline swagger.go:10]: I don't think changing this value may be a good idea.

**Review by ubogdan — CHANGES_REQUESTED** [review]
Please undo the changes regarding the default swagger instance name.

File: swagger.go (hunk 1 above)
Comment (pilinux) [reply swagger.go:10]: `const Name = "swagger"` will generate `docTemplateswagger` and `SwaggerInfoswagger` which is OK from the perspective of a linter, but a bit difficult to read from a human's perspective. That's why I capitalized it. And this change also didn't break any other part of the full repo.

After reading this comment, if you still believe that it is a better idea to keep it as `"swagger"`, kindly let me know. I will revert it back.

Thank you.

Comment (ubogdan) [reply swagger.go:10]: The generates SwaggerInfo if Instance name == "swagger" and SwaggerInfo+InstanceName if not.
```
var SwaggerInfo{{ if ne .InstanceName "Swagger" }}{{ .InstanceName }} {{- end }} = &swag.Spec{
```

Please roll back that change.

**Comment by ubogdan** [comment]
Please update the doc of all examples with the latest "genetator version"

File: swagger.go (hunk 1 above)
Comment (pilinux) [reply swagger.go]: done.

File: gen/gen.go [hunk 2]
```diff
@@ -407,21 +407,21 @@ package {{.PackageName}}
 
 import "github.com/swaggo/swag"
 
-const docTemplate_{{ .InstanceName }} = ` + "`{{ printDoc .Doc}}`" + `
+const docTemplate{{ .InstanceName }} = ` + "`{{ printDoc .Doc}}`" + `
```
Comment (ubogdan) [inline gen/gen.go:411]: Ok then let's update this to 
```
const docTemplate{{ if ne .InstanceName "swagger" }}{{ .InstanceName }} {{- end }} = ` + "`{{ printDoc .Doc}}`" + `
```

File: gen/gen.go [hunk 3]
```diff
@@ -407,21 +407,21 @@ package {{.PackageName}}
...
+var SwaggerInfo{{ if ne .InstanceName "swagger" }}{{ .InstanceName }} {{- end }} = &swag.Spec{
 	Version:     {{ printf "%q" .Version}},
 	Host:        {{ printf "%q" .Host}},
 	BasePath:    {{ printf "%q" .BasePath}},
 	Schemes:     []string{ {{ range $index, $schema := .Schemes}}{{if gt $index 0}},{{end}}{{printf "%q" $schema}}{{end}} },
 	Title:       {{ printf "%q" .Title}},
 	Description: {{ printf "%q" .Description}},
 	InfoInstanceName: {{ printf "%q" .InstanceName }},
-	SwaggerTemplate: docTemplate_{{ .InstanceName }},
+	SwaggerTemplate: docTemplate{{ .InstanceName }},
```
Comment (ubogdan) [inline gen/gen.go:421]: and this to 
```
	SwaggerTemplate: docTemplate{{ if ne .InstanceName "swagger" }}{{ .InstanceName }} {{- end }}
```

**Review by ubogdan — CHANGES_REQUESTED** [review]
I see what you mean now. Please update according to the changes and regenerate docs once again.
Thanks for your patience.

**Comment by ubogdan** [comment]
@pilinux it looks I'm not in very good shape this week. So please bear with me.

File: gen/gen.go (hunk 2 above)
Comment (pilinux) [reply gen/gen.go:411]: Done.

File: gen/gen.go (hunk 3 above)
Comment (pilinux) [reply gen/gen.go]: Done.

**Review by ubogdan — APPROVED** [review]
LGTM

**Comment by ubogdan** [comment]
@pilinux Thanks for your contribution.

**Pull request merged**